daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -127,46 +126,6 @@ class TransactionCoderSpec
       }
     }
 
-    "do transactions with default versions" in {
```

Comment:
Since #6365, default version is deprecated. no need to bother any more with this test.